src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+
+## Vision - Cameras and Map
+
+The AI is not a biological entity, and it’s only barely a “physical” one. It doesn’t need to care about specifics of how things look, but rather whether or not an area is intact, safe, and secure. As such, the vision system is an optional ability that an AI player can shut down, whether to free space up for something else, or as an emergency measure.
+
+### Map Vision
+
+The “map” view replaces the entirety of the AI player's vision if the camera ability is shut down, or if there’s no cameras *and* they activate a composite vision ability. It should provide information about the station’s hull - walls, floors (or a glaring lack of one or being stripped down to a lattice), windows. It also should display machines that have an AI control wire - deemphasizing them if the wire’s been snipped, and potentially not showing them at all if there’s no camera that can see the machine (to make camera snipping actually work around APCs, air alarms). Fire alarms and atmosphere alarm status should potentially be shown over the area too - perhaps as an ability.
+
+While using the AI map vision, the crew can be seen via suit sensors, only if they have tracking on. Otherwise AI is unable to tell if someone is in the area without using cameras. A potential ability idea is to give cameras a motion detection cone that gets rendered for AI, and highlight entities within it - differently from tracked crew that enters it.
+
+The map vision should not use “#00FF00 on #000000” color scheme for purpose of reducing eye fatigue. Consider dark dark gray background as a default for areas that are not space exposed. Preferably implement a way to set custom colors for your UI.
+
+### Cameras
+
+Cameras cover almost all of the crew areas, places where most people work and exist within, to monitor their wellbeing and the state of the station. They should not be nearly free to build, and should require an autolathe producible board, some steel, LV wire.
+
+They should not be more durable than a person - a hard enough knock should deactivate it for a small period of time, while sustaining enough damage will keep it offline until repaired. A non-violent option for keeping cameras offline should be available, using cloth and/or gauze makes it impossible to see through a camera, taking a relatively long time to cover it completely (3 or 5 second doafter, balanced by it being a “stealth” option). A camera disabled in such a way should still be shown as active for AI and warden. Though it should have no motion detection cone and convey such explicitly.
```

Comment:
Should there be reinforced cameras or emplaced wall cameras for things like Armory/Perma/AI Core? They would have a reduced vision code (as they wouldn't be mounted one tile out from a wall and wouldn't be able to see behind them/in a wider cone as such). These would be much harder to break, and might be more difficult to cover up.